source/arche/tutorial/setup.rst

```diff
@@ -62,7 +62,7 @@ Make a new input file that is a copy of the test input file
 
```

Comment:
All sed -i'' "s/... commands didn't work for me (lines 32, 33, 34 and 70). I had to add a space between the -i and the quotation marks to make it work.
For example, for line 32, I wrote
$ sed -i '' "s/tutorial_facility/storage/g" src/storage*